samples/dymaptic.GeoBlazor.Core.Sample.TokenRefresh/dymaptic.GeoBlazor.Core.Sample.TokenRefresh.Client/Pages/Home.razor

```diff
@@ -0,0 +1,132 @@
+﻿@page "/"
+@rendermode InteractiveAuto
+
+<PageTitle>GeoBlazor Token Refresh Sample</PageTitle>
+
+<div class="container-fluid">
+    <div class="row">
+        <div class="col-12">
+            <h1 class="mb-4">GeoBlazor Token Refresh Sample</h1>
+            
+            <!-- Documentation Section -->
+            <div class="documentation-section mb-4">
+                <p><strong>Token Authentication Process</strong></p>
+                <p>
+                    <strong>This sample demonstrates how to implement automatic token refresh in GeoBlazor applications 
+                    using server-side authentication with ArcGIS Portal or ArcGIS Online.</strong>  
+                    For more details on setting up OAuth credentials in Portal or ArcGIS Online please refer to the following tutorial: 
+                    <a href="https://developers.arcgis.com/documentation/security-and-authentication/app-authentication/tutorials/create-oauth-credentials-app-auth" target="_blank">Create OAuth credentials for app authentication</a>
```

Comment:
This should be pointing users to https://docs.geoblazor.com/pages/appTokenRefresh.html, which needs to be updated to match the code in this sample (so we should also have a GeoBlazor Pro PR to update `docs/pages/appTokenRefresh.md`). If you want to point to the Esri doc, I would do it from within that GeoBlazor doc page.